src/api/useSearchVenuesOffer/useSearchVenueOffers.ts

```diff
@@ -0,0 +1,134 @@
+import { SearchResponse } from '@algolia/client-search'
+import { flatten } from 'lodash'
+import { useMemo, useRef } from 'react'
+import { InfiniteQueryObserverOptions, useInfiniteQuery } from 'react-query'
+
+import { VenueListItem } from 'features/offer/components/VenueSelectionList/VenueSelectionList'
+import { useIsUserUnderage } from 'features/profile/helpers/useIsUserUnderage'
+import { initialSearchState } from 'features/search/context/reducer'
+import { formatFullAddressStartsWithPostalCode } from 'libs/address/useFormatFullAddress'
+import { AlgoliaHit, Geoloc } from 'libs/algolia'
+import { useSearchAnalyticsState } from 'libs/algolia/analytics/SearchAnalyticsWrapper'
+import { fetchOffers } from 'libs/algolia/fetchAlgolia/fetchOffers'
+import { useTransformOfferHits } from 'libs/algolia/fetchAlgolia/transformOfferHit'
+import { Position } from 'libs/geolocation'
+import { formatDistance } from 'libs/parsers'
+import { QueryKeys } from 'libs/queryKeys'
+import { getNextPageParam } from 'shared/getNextPageParam/getNextPageParam'
+import { Offer } from 'shared/offer/types'
+
+export type UseSearchVenueOffersType = {
```

Comment:
Un détail mais ce serait mieux quand même, sinon on a l'impression que tu types la méthode, pas les options.
```suggestion
export type UseSearchVenueOffersOptions = {
```